Source/ParameterEncoding.swift

```diff
@@ -85,14 +85,19 @@ public struct URLEncoding: ParameterEncoding {
         case brackets
         /// No brackets are appended. The key is encoded as is.
         case noBrackets
+        
+        case index
```

Comment:
Please add inline documentation, similar to the documentation for the other cases.